Cargo.toml

```diff
@@ -274,6 +274,7 @@ cranelift-fuzzgen = { path = "cranelift/fuzzgen" }
 cranelift-bforest = { path = "cranelift/bforest", version = "0.119.0" }
 cranelift-bitset = { path = "cranelift/bitset", version = "0.119.0" }
 cranelift-control = { path = "cranelift/control", version = "0.119.0" }
+cranelift-srcgen = { path = "cranelift/srcgen" }
```

Comment:
It felt to me that this kind of crate isn't the same as the other Cranelift crates in terms of versioning, but I suspect we will have to give it a `0.119.0` version just to get it to pass the `publish.rs` script? I noticed that some fuzzing infrastructure crate had a `0.0.0` version but perhaps that isn't published?